com.sudovanilla.penpot-desktop.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: com.sudovanilla.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://sudovanilla.com/distribute/applications/penpot-desktop/latest/Penpot-Desktop.AppImage
+        sha256: d448a70f45cef03a89efe2f5c28b1dae2a0a53cf627749865195bec422e7493a
+        only-arches: [x86_64]
+      - type: file
+        url: https://sudovanilla.com/distribute/applications/penpot-desktop/latest/Penpot Desktop-arm64.AppImage
+        sha256: d59c26d66344e90f8cdb58b5468adfb8b8d16364da25a643c39deff873f421c3
+        only-arches: [arm64]
+      - type: file
+        path: ./run.sh
+      - type: file
+        path: ./penpot-desktop.desktop
+      - type: file
+        path: com.sudovanilla.penpot-deskop.appdata.xml
+    build-commands:
+      - install -Dm755 ./run.sh /app/bin/run.sh
+      - mv penpot-desktop.png /app/share/icons/hicolor/512x512/apps/com.sudovanilla.penpot-desktop.png
+      - chmod +x Penpot*.AppImage
+      - ./Penpot*.AppImage --appimage-extract
+      - mv squashfs-root /app/bin/penpot-desktop
```

Comment:
in this squashfs-root there is a .desktop and an icon. These are the file have been asking that it be used, with `desltop-file-edit` for adapt the .desktop file (exec and icon).

Not sure what was unclear when I said to install the files from the appimage.